code/modules/customitems/item_defines.dm

```diff
@@ -2380,3 +2135,91 @@ All custom items with worn sprites must follow the contained sprite system: http
 	..()
+
+
+/obj/item/clothing/suit/storage/toggle/fluff/leonid_chokha //Old Rebel's Chokha - Leonid Myagmar - lucaken
+	name = "old rebel's chokha"
+	desc = "A not-so traditional Vysokan Chokha made out of beat-up gurmori leathers, worn-out to the point of seeming ancient. Though it might have been a Host-boy's garment once, it is now \
+	swarmed with dozens of patches all with varying colours and origins - the most prominent of which is a large image of a crimson snake on a white plain, curled around a fire. It's the oldest \
+	out of all the adornments."
+	icon = 'icons/obj/custom_items/leonid_chokha.dmi'
+	icon_override = 'icons/obj/custom_items/leonid_chokha.dmi'
+	icon_state = "leonid_chokha"
+	item_state = "leonid_chokha"
+	contained_sprite = TRUE
+
+
+/obj/item/clothing/suit/storage/toggle/fluff/sezrak_coat //red Domelkoan Coat - Sezrak Han'san - captaingecko
+	name = "red Domelkoan Coa"
+	desc = "A warm coat made in Domelkos. This red coat is stuffed with yupmi fur and made out of reinforced cloth-like synthetic materials, both to keep the wearer warm in the cold winters of \
+	Moroz, and to resist all but the rougher treatments... All the while remaining good-looking enough. Both shoulders on this coat feature the standard of the Han'san clan-house, presented in a \
+	gilded color."
+	icon = 'icons/obj/custom_items/sezrak_coat.dmi'
+	icon_override = 'icons/obj/custom_items/sezrak_coat.dmi'
+	icon_state = "sezcoat"
+	item_state = "sezcoat"
+	contained_sprite = TRUE
+
+/obj/item/clothing/accessory/poncho/fluff/sezrak_cape //Red Han'san Cape - Sezrak Han'san - captaingecko
+	name = "red Han'san cape"
+	desc = "This is a cape loosely based on the style of Dominian nobility, the latest fashion across Dominian space, although it doesn't feature any of the colors belonging to the Great Houses, \
+	and doesn't bear the symbolism of the ones worn by Tribunalist priests. The left shoulder-side bears the standards of the Han'san clan-house with a small, discreet symbol of gilded colors, \
+	instead of the usual Green used for this house."
+	icon = 'icons/obj/custom_items/sezrak_coat.dmi'
+	icon_override = 'icons/obj/custom_items/sezrak_coat.dmi'
+	icon_state = "sez_cape"
+	item_state = "sez_cape"
+	contained_sprite = TRUE
+
+/obj/item/journal/fluff/mrakiizar_book //Worn Journal - Azradov Mrakiizar - kingoftheping
+	name = "worn journal"
+	desc = "A heavily worn journal-like hardcover book. It is filled with lots of handwritten notes, lists and some sketches in between. Both the contents in the book, aswell as the loose pages \
+	tucked in between are mostly written in Siik'maas. The front has a strip of tape with the name 'Ahkrraazarjhri Maalhalkasanurran' on it."
+	closed_desc = "A heavily worn journal-like hardcover book. It is filled with lots of handwritten notes, lists and some sketches in between. Both the contents in the book, aswell as the loose \
```

Comment:
`closed_desc` should not be set, as that description is added on top of the actual desc when the journal is opened. An example of this can be seen with Kathira's journal.